deckhouse-controller/entrypoint.sh

```diff
@@ -34,4 +34,37 @@ EOF
 
 cat ${MODULES_DIR}/values-${bundles_map[$bundle]}.yaml >> ${MODULES_DIR}/values.yaml
 
-exec /sbin/tini -- /usr/bin/deckhouse-controller start
+set +o pipefail
+set +e
+
+PID=0
+EXITCODE=0
+
+signal_handler() {
+  echo "{\"level\":\"warning\", \"msg\": \"Catch signal ${1}\"}"
+  case "${1}" in
+  "SIGUSR1" | "SIGUSR2")
+    kill "${PID}"
+    wait "${PID}"
+    run_deckhouse
+    ;;
+  *)
+    kill -"${1}" "${PID}"
+    ;;
+  esac
+}
+
+run_deckhouse() {
+  /sbin/tini -s -- /usr/bin/deckhouse-controller start &
```

Comment:
I'd like to use tini as a main supervisor to run entrypoint.sh, but not vise versa